spec/mailers/anonymized_data_download_mailer_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
+
+RSpec.describe AnonymizedDataDownloadMailer, type: :mailer do
```

Comment:
We should add specs here or delete this file. Typically in mailer specs you'd just check to make sure the subject and to/from are being set correctly

Reply:
Agreed. I will fill out the specs (and add more custom specs) in parallel.